test/linter.spec.js

```diff
@@ -0,0 +1,30 @@
+const utils = require('./helpers/utils'),
+  { mockCommand } = require('./helpers/mocks'),
+  { runCommand } = require('../generators/app/command'),
+  { exampleProjects } = require('./helpers/constants'),
+  commands = utils.getCommands();
+
+describe.each(exampleProjects)('Example project with %s', (projectName, { kickoffOptions }) => {
+  beforeAll(() => {
+    mockCommand();
+    return utils.runKickoff({
+      ...kickoffOptions,
+      nodeVersion: '8.9.12',
+      npmVersion: '6.4.1',
+      inTraining: false,
+      projectName: 'Project',
+      projectDescription: 'Project',
+      urlRepository: 'https://test.com.ar'
+    });
+  });
+
+  test.each(commands)('run the EsLinter for each project generated', command => {
+    jest.setTimeout(15000);
+    const directory = utils.getTestDirectory('Project');
+    return runCommand({ ...command, spawnOptions: { cwd: directory } }).then(res => {
+      const errors = unescape(res);
+      const problems = errors.search('✖ 1 problem');
```

Comment:
If the linter detects some errors, I think it finishes the execution with an error, you should check for that instead of parsing the whole response. 
Also it should be nice to display the linter errors.

Reply:
* the error will always show in the terminal
[
![image](https://user-images.githubusercontent.com/52759744/67781138-0bd17d80-fa35-11e9-88d1-d5448d7818bb.png)
](url)